### ISSUE TITLE
Support JPA Filtering

### ISSUE THREAD

**Issue by danielpetisme** [jhipster/jhipster-quarkus#41]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by joewhite101** [comment]
This is related to https://www.jhipster.tech/entities-filtering/

What is the general thinking on this one? I like the JHipster filter format. It is quite powerful and very quick to understand. It would be cool if both quarkus and springboot jhipster services exposed the same filter format even if the underlying query structure is different.

I wonder if it makes sense to map the JHipster filter structure to Panache queries? Other ideas? Or is this not totally needed? 

cc @jdubois

**Comment by jdubois** [comment]
Indeed, that's a cool feature we have thanks to Spring Data JPA - I guess you can have something similar with Panache, but I'm not sure. Not everybody uses it, so I wouldn't say it's mandatory, but that's definitely added value. Also, this would cause some issues on our front-ends as they expect this behavior from the backend.
